packages/dds/tree/src/test/shared-tree/fuzzEditGenerator.ts

```diff
@@ -192,7 +199,7 @@ function getRandomPlace(tree: ISharedTree, random: IRandom): UpPath {
 	}
 }
 
-function getExistingRandomNodePosition(tree: ISharedTree, random: IRandom): UpPath {
+function getExistingRandomNodePosition(tree: ISharedTree, random: IRandom): PathWithNodeRange {
```

Comment:
I can see how changing the return type here was the simplest way to get what we want, but it makes the contract for `getExistingRandomNodePosition` a little weird. In fact, it seems like after such a change the function would be better named `getExistingRandomNodeRangePath`.

I agree we don't want to duplicate the functions though. So, what I propose is to rename this function `getExistingRandomNodeRangePath` and create a new `getExistingRandomNodePosition` function that has the same signature as before, but is implemented on top of `getExistingRandomNodeRangePath`.

Reply:
renamed `getExistingRandomNodePosition` to `getExistingRandomNodeRangePath`, created new `getExistingRandomNodePosition` function implemented on top of `getExistingRandomNodeRangePath`